doc/how_to_fit.rst

```diff
@@ -0,0 +1,68 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _how_to_fit:
+
+##########################################
+How to use PyPSA-Earth for your energy problem?
+##########################################
+
+PyPSA-Earth can be tailored to represent any part of the world quite easily. The following procedure may be recommended.
+
+1. Adjust the model configuration
+=========================================
+
+The main parameters needed to customize the inputs for your national-specific data are defined in the :ref:`configuration <config>` file `config.yaml`. The configuration settings should be adjusted according to a particular problem you are intended to model. The main regional-dependent parameters are:
+
+* `countries` parameter which defines a set of the countries to be included into the model;
+
+* `cutouts` and `cutout` parameters which refer to a name of the climate data archive (so called `cutout <https://atlite.readthedocs.io/en/latest/ref_api.html#cutout>`_) to be used for calculation of the renewable potential.
+
+Apart of that, it's worth to check that there is a proper match between the temporal and spatial parameters across the configuration file as it is essential to build the model properly. Generally, if there are any mysterious error message appearing during the first model run, there are chances that it can be resolved by a simple config check.
+
+It could be helpful to keep in mind the following points:
+
+1. the cutout name should be the same across the whole configuration file (there are several entries, one under under `atlite` and some under each of the `renewable` parameters);
+
+2. the countries of interest defined with `countries` list in the `config.yaml` should be covered by the cutout area;
+
+3. the cutout time dimension, the weather year used for demand modeling and the actual snapshot should match.
+
+2. Load all the common data
```

Comment:
This may be confusing, I'd remove this part as the workflow takes care of this.
It seems that to run the model there is the need to load all the common data first, which is not the case

Reply:
Agree. How to remove the common data should be clear from the installation instructions